packages/cozy-harvest-lib/README.md

```diff
@@ -69,6 +69,28 @@ Cozy-Harvest-Lib provides a defined range of field labels which do not need to b
 
 Any konnector can provide new locales for this field labels or add new ones. They will be loaded by the AccountForm component.
 
+#### `placeholder`
+
+We could defined a localized placeholder in the manifest
+
+They could be defined like this :
+```js
+{
+  ...
+  locales: {
+    𝘭𝘢𝘯𝘨: {
+      fields:
+        𝘧𝘪𝘦𝘭𝘥𝘯𝘢𝘮𝘦: {
```

Comment:
What is this font? Is it normal?

Reply:
This is an italic font , because you can't set "italic markdown information" in "markdown code"

Reply:
Maybe

```js
{
  locales: {
    [lang]: {
      fields: {
        [fieldName]: {
```
is clearer ? It is JS-compatible as well. What do you think ?